### ISSUE TITLE
Can't comment on issue

### ISSUE THREAD

**Issue by jpmeijers** [Ideetron/Lorank#19]
I want to provide further details on an issue, but it's locked and I can't comment.

https://github.com/Ideetron/Larank/issues/16#issuecomment-828788103_

**Comment by abhatikar** [comment]
@jpmeijers  reply [here](https://github.com/Ideetron/Larank/issues/18#issue-870401385)

**Comment by jpmeijers** [comment]
Regarding your question on #16 I would recommend you look at https://github.com/danielknox/ttn-resin-gateway-bbb. Ever since I struggled with #16 and couldn't really get it to work satisfactory, I've used this resin based installation. And I haven't looked back! It would have been nice if @devlaam @barthiddink supported the software for their gateway a little better.

**Comment by barthiddink** [comment]
@jpmeijers, Devlaam and I are working hard to give everybody the best experience for their money. We have thousands of very happy customers all around the world. Baking a new BBG isn't that hard. Don't forget the HW modification after you buy a new BBG. And JP, don't forget how you came by your Lorank8 in the first place! :)

**Comment by jpmeijers** [comment]
Yes, but I'm just here trying to help others. Closing a ticket (#16, the original issue) doesn't help with that.

**Comment by abhatikar** [comment]
@barthiddink What modification you need to do after buying a new BBG ?
